arangod/Aql/RestAqlHandler.cpp

```diff
@@ -709,6 +710,11 @@ RestStatus RestAqlHandler::handleUseQuery(std::string const& operation,
 
   auto const rootNodeType = _engine->root()->getPlanNode()->getType();
 
+  VPackOptions const* opts = &VPackOptions::Defaults;
+  if (_engine) {  // might be destroyed on shutdown
```

Comment:
Do we need this check? Below it's used without this condition.
Or it's not needed only for `operation == StaticStrings::AqlRemoteExecute`?

Reply:
It is probably true that `_engine` is always != nullptr here, but I am not 100% sure. I think the check doesn't do any harm, so I would prefer to leave it in.

Reply:
Ok